tennis-scheduler-web-app/src/components/TennisCourts.jsx

```diff
@@ -1,9 +1,44 @@
-import React from "react"
+import React, { useState, useEffect } from "react";
+import RolandGarrosCentral from "../assets/RolandGarrosCentral.jpg";
+import { TennisCourtInfo } from "./TennisCourtInfo";
+import "../styles/courts.css";
+import { getTennisCourts } from "../api/TennisCourtApi";
 
 export const TennisCourts = () => {
+  const [tennisCourts, setTennisCourts] = useState([]);
 
-    return (
-        <> </>
-    )
-}
+  useEffect(() => {
+    getCourts();
+  }, []);
 
+  async function getCourts() {
+    getTennisCourts()
+      .then((data) => {
+        setTennisCourts(data.data);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
It would be nice to show user that there was an error, on screen not in console

Reply:
Also, use full length names -> error